models2/Preferences.es6.js

```diff
@@ -0,0 +1,118 @@
+import RedditModel from './RedditModel';
+const T = RedditModel.Types;
+
+export default class PreferencesModel extends RedditModel {
+  static type = 'preferences';
+
+  static PROPERTIES = {
+    affiliateLinks: T.bool,
+    allowClicktracking: T.bool,
+    beta: T.bool,
+    clickgadget: T.bool,
+    collapseReadMessages: T.bool,
+    compress: T.bool,
+    credditAutorenew: T.bool,
+    defaultCommentSort: T.string, // It would be nice to have T.oneOf like react here
```

Comment:
It's a good point, I would propose having it take an optional default value, otherwise undefined. I mostly wanted to this comment in place to document future improvements we could do that I was putting off. Do you think its better to remove it?